routes/add_user.js

```diff
@@ -1,20 +1,39 @@
 require('../models/user.js');
+var auth = require("../modules/auth");
 var mongoose = require("mongoose");
 var User = mongoose.model('User');
 
 module.exports = function(req, res) {
-	var infos = req.body;
-  var user = new User({
-	  cip: infos.cip,
-	  prenom: infos.prenom,
-	  nom: infos.nom,
-	  email: infos.email,
-	  concentration: infos.concentration,
-	  promo: infos.promo
-	});
-	
-	user.save(function(err) {
-    if (err) throw err;
-    res.redirect('/');
-  });
+    var infos = req.body;
+    User.count({}, function (err, count) {
```

Comment:
it's ok for dev for now, we will setup a dev environment to avoid code for dev envrionement only